provider/terraform/website/docs/version_2_upgrade.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: "google"
+page_title: "Terraform Google Provider Version 2 Upgrade Guide"
+sidebar_current: "docs-google-provider-version-2-upgrade"
+description: |-
+  Terraform Google Provider Version 2 Upgrade Guide
+---
+
+# Terraform Google Provider Version 2 Upgrade Guide
+
+~> **NOTE:** This upgrade guide is a work in progress and will not be completed until the release of version 2.0.0 of the provider later this year. Many of the topics discussed, except for the actual provider upgrade, can be performed using the most recent 1.X version of the provider.
```

Comment:
Our docs are already set up for `2.0.0` release, so we can probably remove the note about this being a work in progress here; unless we're planning to cut a website release, this will be released alongside `2.0.0`.